src/app/components/footer/index.js

```diff
@@ -67,56 +69,62 @@ const Footer = React.createClass({
         break;
       case 'events/event':
         return <EventsSubscription />;
-        break; 
+        break;
       default:
         return <Subscription />;
     }
   },
   renderSocialMediaChannel(url, channel) {
-    return <li key={channel} className={classnames('channel', channel)}>
-      <a
-        href={url}
-        onClick={this.onClickSocial(channel)}
-        target="_blank"
-      >
-        <SVG
-          className="logo"
-          title={`${channel} logo`}
-          spritemapID={channel}
-        />
-      </a>
-    </li>;
+    return (
+      <li key={channel} className={classnames('channel', channel)}>
+        <a
+          href={url}
+          onClick={this.onClickSocial(channel)}
+          target="_blank"
```

Comment:
Add `rel="noopener noreferrer"` to external links. [Source](https://medium.com/@jitbit/target-blank-the-most-underestimated-vulnerability-ever-96e328301f4c#.mc9ngax9v)